nfafrontend/src/app/shared/data-storage.service.ts

```diff
@@ -13,12 +14,11 @@ export class DataStorageService {
     console.log(nfa);
    return this.http.post('http://localhost:8080/nfabackend/webapi/nfa_catalog/' , nfa);
   }
-  storeProject(newproject: Project) {
+  storeProject(newproject: Project, type: Type[]) {
     console.log(newproject);
-    return this.http.post('http://localhost:8080/nfabackend/webapi/project/create', newproject);
+    return this.http.post('http://localhost:8080/nfabackend/webapi/project/create', newproject:type);
```

Comment:
I get some error here saying - 
ERROR in src/app/shared/data-storage.service.ts(19,95): error TS1005: ',' expected.

Reply:
I have the same error. @maielgendy Please try to check this out as a new repository and see if it builds then.